buildSrc/src/main/java/com/google/firebase/gradle/ReleaseGenerator.kt

```diff
@@ -101,25 +141,49 @@ open class ReleaseGenerator : DefaultTask() {
   ) =
     repo
       .log()
-      .addPath("$directory/")
+      .addPath("$directory/CHANGELOG.md")
       .addRange(previousReleaseRef, currentReleaseRef)
       .setMaxCount(1)
       .call()
       .iterator()
       .hasNext()
 
+  private fun getDirChanges(
+    repo: Git,
+    previousReleaseRef: ObjectId,
+    currentReleaseRef: ObjectId,
+    directory: String
+  ) =
+    repo
+      .log()
+      .addPath("$directory/CHANGELOG.md")
+      .addRange(previousReleaseRef, currentReleaseRef)
+      .call()
+      .map { toCommitDiff(repo, it) }
+      .toList()
```

Comment:
Shouldn't need the `.toList()`, since you're mapping an `Iterable` to a single value- the return will be a `List` anyhow.